src/common/cli_wrapper.cpp

```diff
@@ -128,6 +128,38 @@ void CLIWrapper::parse(int argc, char **argv) {
   }
 }
 
+void CLIWrapper::parseAliases() {
```

Comment:
Since this is a new functionality, can you add more comments about this function and also inside the function?

Reply:
Function documentation was in the header file. Added more comments inside the function.